.github/workflows/pypi.yml

```diff
@@ -19,17 +19,22 @@ jobs:
         platform:
           - runner: ubuntu-22.04
             target: x86_64
+            manylinux: auto
           - runner: ubuntu-22.04
             target: x86
-          # FUBAR
-          # - runner: ubuntu-22.04
-          #   target: aarch64
+            manylinux: auto
+          - runner: ubuntu-24.04-arm
```

Comment:
Technically I think this isn't needed, since cross-compiling works. But running on an ARM runner should be slightly faster anyways?